### DISCUSSION TITLE
按shift+P键没有选项跳出，关系图谱也没有办法调出了

### DISCUSSION THREAD

**Discussion by MrLittleShark** [MuiseDestiny/zotero-style#248] [General]
是不是bug了
![Snipaste_2023-03-14_15-31-59](https://user-images.githubusercontent.com/112299157/224927442-8923c376-e57e-48dd-91ed-38c38832b2c7.jpg)

**Comment by MuiseDestiny** [comment]
更新试试呢

**Reply by MrLittleShark** [reply]
嗯嗯呢，用上新版本啦！关于图谱说一下一些使用体验吧：1. 我电脑一般接扩展屏，在扩展屏上，那个粉色图标会大变形。 2. 图谱的高度不可调。 3. 关于节点的问题，节点上没有标注，只有放到很大才会有标注，这样用起来似乎会有些不太方便诶， 不过还是谢谢你！开发好用的插件。

![1](https://user-images.githubusercontent.com/112299157/224943867-782f94df-bd7b-475c-a564-4dd0255eb999.jpg)

**Reply by MuiseDestiny** [reply]
问题1似乎解决了；问题2以后版本会加入调节的，现在只是写了核心功能；问题3已解决。

**Reply by MuiseDestiny** [reply]
重新在github下载最新版安装。

**Comment by Guanrhchn** [comment]
您好，我也遇到了相同的问题。shift+p没有可选功能，zotero和插件都是最新版本
